src/format.js

```diff
@@ -85,9 +85,10 @@ function extractError (trace, span) {
   const error = span.context()._tags['error']
 
   if (error instanceof Error) {
-    trace.meta['error.msg'] = error.message
-    trace.meta['error.type'] = error.name
-    trace.meta['error.stack'] = error.stack
+    trace.meta['sfx.error.message'] = error.message
+    trace.meta['sfx.error.object'] = error.name
+    trace.meta['sfx.error.kind'] = error.name
```

Comment:
Is the redundant kind tag necessary?  I see `error.kind` is considered optional in OT, though error.object should be the instance itself (`error.toString()`)?

Reply:
I'm not entirely sure yet so I put both here for now until I figure it out. `error.toString()` at first seems to be just concatination of `kind` and `message` and I'm not sure if that's right. In Java at least, `kind` seems to be the error/exception type name while as object is the full qualified path but I'm not 100% sure about this yet either. Need to look deeper into this and figure out what to use exactly.

Reply:
error.object is supposed to be the type of the exception/error object.  In Java that's like java.lang.IllegalArgumentException.  error.kind is supposed to be a shorter version of this (if possible - in the Java example it would be just IllegalArgumentException).    For node.js I think these will mostly be Error anyway so it's find to have them identical.

Reply:
@johnbley modeling [from these](https://opentracing.io/specification/conventions/) (Captured errors section) the error.object should be the actual instance, whose utility seems pretty limited to certain platforms.  I think what you've done in java makes sense there, but maybe not in node?

I may have lost the thread but was it decided the redundant kind is necessary?

Reply:
As far as I understand, `kind` doesn't seem redundant to me. `kind` is required and should always be there. `object` may or may not depending on the platform. Node does not have something that translates to "object" as in Java or Python. I'll omit object.